client/me/profile/index.jsx

```diff
@@ -169,11 +145,20 @@ class Profile extends Component {
 							/>
 						</FormFieldset>
 
+						<p className="profile__public-url">
+							{ this.props.translate( 'View your public profile at {{a}}{{url/}}{{/a}}.', {
+								components: {
+									a: <ExternalLink href={ relativeProfileUrl } />,
```

Comment:
Is this expected to change this to an external link? Now it requires a full page reload even though technically the page is still within Calypso.

Reply:
Yep. The profile page should open in a new tab so that state of the profile settings page is maintained.

We have a prompt if the user leaves the page with unsaved settings, but we’re then forcing them to choose whether to save changes before they can see the profile. I should think most people click this link out of curiosity and don’t deliberately come here to access their profile.